desktop/flipper-ui-core/src/fb-stubs/pluginMarketplaceAPI.tsx

```diff
@@ -7,10 +7,10 @@
  * @format
  */
 
-export async function loadPluginsFromMarketplace() {
-  // Marketplace is not implemented in public version of Flipper
-}
+import {MarketplacePluginDetails} from '../reducers/plugins';
 
-export default () => {
-  // Marketplace is not implemented in public version of Flipper
-};
+export async function loadAvailablePlugins(): Promise<
+  MarketplacePluginDetails[]
```

Comment:
Would it be possible to provide a minimal schema of what is required fields for plugin to be detected and downloadable from the marketplace API?

Current typing also including fields for docs `publishedDocs`, which I believe is only useful for internal use in Meta/FB.

I would like to provide some basic schema, so others can create their own API easily to feed Flipper with plugins.

In general, the main object is basically:
```js
type MarketPlacePlugin = {
   ...package.json,
// flipper specific fields
specVersion: number; // not sure how to get this, since not present in package.json
availableVersions: [], // array of available plugins (type MarketPlacePlugin)
  downloadUrl: string; // provide a link to a tarball
  lastUpdated: Date; // need provide date from last published package
// those bellow, not sure if just keep as it is by default?
 isActivatable: false;
  isBundled: false;
  // Indicates whether plugin should be enabled by default for new users
  isEnabledByDefault: boolean;
  // Determines whether the plugin is a part of the Flipper JS bundle.
  isBundled: boolean;
  // Determines whether the plugin is physically available for activation in Flipper.
  isActivatable: boolean;
}
```